packages/cxl-ui/src/components/cxl-dashboard-header.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/progress-bar';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlDashboardHeaderGlobalStyles from '../styles/global/cxl-dashboard-header-css.js';
+import cxlDashboardHeaderStyles from '../styles/cxl-dashboard-header-css.js';
+
+@customElement('cxl-dashboard-header')
+export class CXLDashboardHeaderElement extends LitElement {
+  static get styles() {
+    return [cxlDashboardHeaderStyles];
+  }
+
+  @property({ type: String }) title = 'Hello';
+
+  @property({ type: String }) subtitle = 'My dashboard';
+
+  @property({ type: String }) name = '';
+
+  @property({ type: String }) notificationCount = '6';
+
+  @property({ type: String }) updateTitle = "What's new in CXL";
+
+  @property({ type: String }) lastCourseTitle = 'Advanced experimentation analysis';
+
+  @property({ type: String }) lastCourseSubitle = 'Continue where you left off';
+
+  @property({ type: String }) progressbarTitle = 'Completed 2 lessons of 6 lessons';
+
+  @property({ type: String }) progressbarValue = '0.34';
+
+  @property({ type: String }) cta = 'Browse all courses';
+
+  @property({ type: String }) ctaLink = '';
+
+  @property({ type: String }) cta1 = 'Browse fast skills';
+
+  @property({ type: String }) cta1Link = '';
+
+  @property({ type: String }) cta2 = 'Create your personal learning roadmap';
```

Comment:
As I understand  this is the second button and roadmap button is the 3rd, so this can't be used as such CTA.